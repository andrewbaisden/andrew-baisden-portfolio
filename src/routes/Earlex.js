import React from 'react';
import { Helmet } from 'react-helmet';
import Header from '../components/Header';
import Footer from '../components/Footer';
import PortfolioPageEarlex from '../components/PortfolioPageEarlex';
import CheckPortfolioPage from '../components/CheckPortfolioPage';
import PortfolioCarouselEarlex from '../components/PortfolioCarouselEarlex';

const Earlex = () => (
  <div>
    <div>
      <Helmet>
        <meta
          name="description"
          itemProp="description"
          content="I was hired as a Graphic Designer in a temporary position. I was working with two other Graphic Designers as well as a few other departments."
        />
        <title>Earlex - Portfolio - Full Stack Developer | Andrew Baisden | London</title>
        <link rel="canonical" href="https://andrewbaisden.com/portfolio/earlex" />
      </Helmet>
    </div>
    <CheckPortfolioPage />
    <Header />
    <PortfolioPageEarlex companyBG="#F4F4F4" />
    <PortfolioCarouselEarlex />
    <Footer />
  </div>
);

export default Earlex;
