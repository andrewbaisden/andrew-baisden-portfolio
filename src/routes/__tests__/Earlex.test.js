import React from 'react';
import { shallow } from 'enzyme';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Earlex from '../Earlex';

Enzyme.configure({ adapter: new Adapter() });

describe('<Earlex />', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<Earlex />);
    expect(wrapper).toMatchSnapshot();
  });
});
