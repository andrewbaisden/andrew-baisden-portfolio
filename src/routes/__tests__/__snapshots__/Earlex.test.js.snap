// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Earlex /> renders and matches the snapshot 1`] = `ShallowWrapper {}`;
